isolate_example/lib/page_three.dart

```diff
@@ -103,25 +105,32 @@ class DataTransferIsolateController extends ChangeNotifier {
 
   void generateOnSecondaryIsolate() {
     if (running) return;
-
+    _runningTest = 3;
```

Comment:
I think there's been a mixup here. The array of bools is gone, which is great, but there are now three different properties tracking whether a test is running: `_runningTest`, `runningTest`, and `running`.

It looks like `runningTest` is being used to guard access to the Controller's field, which is a legit reason to use a getter. `running` doesn't need to be tracked separately, though. It could either disappear (and consumers could check whether `runningTest` was zero, or it could be a getter that did the check itself.